addons/mail/static/src/utils/common/hooks.js

```diff
@@ -45,7 +45,7 @@ export function onExternalClick(refName, cb) {
     let downTarget, upTarget;
     const ref = useRef(refName);
     function onClick(ev) {
-        if (ref.el && !ref.el.contains(ev.target)) {
+        if (ref.el && !ref.el.contains(ev.composedPath()[0])) {
```

Comment:
Is there a bug here? Can't it be fixed separately?

Reply:
This bug is introduced in this PR when we have three overlays together on the page, and we want that external click knows exactly which overlay should be affected and closed. I can make a separate fix, but I can't reproduce it there.